sdk/identity/identity/test/node/msalNodeTestSetup.ts

```diff
@@ -121,12 +121,12 @@ export async function msalNodeTestSetup(
           {
             regex: true,
             target: `x-client-OS=[a-zA-Z0-9]+`,
-            value: `x-client-OS=x-client-OS`,
+            value: `x-client-OS=Sanitized`,
```

Comment:
Maybe double sanitized, because we might be calling the same function twice at 2 different places - one might be getting called from inside of another file?